src/app/docs/faq/page.mdx

```diff
@@ -0,0 +1,105 @@
+
+
+## Is establishing a connection without relays or when offline possible?
+
+Yes.
+When you share a `NodeAddr`s with "direct addresses", then iroh will try to use these addresses to establish a connection with or without a relay.
+If you're in a local network together you can enable [local network discovery](https://docs.rs/iroh/latest/iroh/endpoint/struct.Builder.html#method.discovery_local_network) to help establish connections in LANs even when the `NodeAddr` doesn't contain direct addresses.
+
+
+## How can I control which relay servers iroh connects to?
+
+Iroh will only talk to relay servers that it knows URLs for.
+By default iroh is configured with 3 relay servers from the [default `RelayMap`](https://docs.rs/iroh/latest/iroh/defaults/prod/index.html).
+If you enable `discovery_n0` or other discovery services, then iroh might connect to relay servers discovered that way.
+By chaning iroh's relay mode or relay map you can control the home relay the endpoint connects to, and by wrapping or writing your own `Discovery` service, you gain control over the relay URLs iroh can discover.
+
+
+## What is "Discovery" in iroh and which one should I enable?
+
+For most usage, using [`discovery_n0`](https://docs.rs/iroh/latest/iroh/endpoint/struct.Builder.html#method.discovery_n0) is the best default.
+Discovery helps iroh find ways to connect to a specific Node ID.
+The Node ID on its own can only be used to identify if you're talking to the right recipient, but doesn't tell how to address the recipient on its own.
+Via configured discovery mechanisms, iroh resolves a Node ID to IP addresses and relay URLs that help to actually attempt a connection.
+For more information on available discovery mechanisms, take a look at the [discovery functions](https://docs.rs/iroh/latest/iroh/endpoint/struct.Builder.html#method.discovery_n0) in the `EndpointBuilder`.
+It's also possible to combine multiple discovery mechanisms at once, or write your own.
+We think it's particularly helpful to write application-specific discovery mechanisms that are tailored to an application's need.
+
+
+## How is iroh development funded?
+
+The company behind iroh is number 0.
+It is partly venture capital and partly founder backed (as in: founders have invested their own money).
+Number 0 is healthy and has investors we actually think are a value-add.
+We earn revenue from customers today through https://n0.computer/n0ps, and are re-launching an “AWS for iroh  protocols” in the next few weeks.
+While it will be a service you can pay for, it won’t have special treatment beyond being convenient.
+We rely on iroh remaining open source, and are committed to keeping it that was, including server-side code for relays and DNS discovery.
+
+
+## How do iroh and libp2p compare?
+
+Generally, we've put a lot of effort into making iroh "peer to peer connections that *just work*".
+This means we've kept the scope small: There's no DHT, swarm, or gossipsub.
```

Comment:
Maybe expand on this a little bit, like "Only a direct p2p QUIC connection between two iroh nodes."  (Or do we call them endpoints?  lol)

Reply:
Thanks for the review. I've expanded in this section a bit.